packages/cbioportal-clinical-timeline/src/TimelineTrack.tsx

```diff
@@ -350,20 +357,23 @@ const TimelineItemWithTooltip: React.FunctionComponent<{
                     });
 
                     setTooltipUid(uid);
+
+                    store.setHoveredTooltipUid(uid);
```

Comment:
this means tooltip will only show up on initial entry and won't adjust as user continues to move mouse.  this is necessary in order to allow user to mouse INTO a tooltip